roles/alloy/defaults/main.yml

```diff
@@ -1,39 +1,39 @@
-version: "1.0.0"
+grafana_alloy_version: "1.0.0"
 
-arch_mapping:
+grafana_alloy_arch_mapping:
   x86_64: amd64
   aarch64: arm64
   armv7l: armhf
   i386: i386
   ppc64le: ppc64le
 
-arch: "{{ arch_mapping[ansible_architecture] | default('amd64') }}"
+grafana_alloy_arch: "{{ grafnana_alloy_arch_mapping[ansible_architecture] | default('amd64') }}"
```

Comment:
Typo in `grafnana_alloy`. Perhaps a correction is not necessary after @gardar's comment.

Reply:
Thanks, got that too